app/lib/ctc_signup_message.rb

```diff
@@ -38,4 +38,37 @@ def self.send(count)
       ctc_signup.touch(:beta_email_sent_at)
     end
   end
-end
\ No newline at end of file
+
+  def self.send_launch_announcement(count)
+    ctc_signups = CtcSignup.where(launch_announcement_sent_at: nil).take(count)
+    ctc_signups.each do |ctc_signup|
+      _send_one_launch_announcement(ctc_signup)
+    end
+  end
+
+  def self._send_one_launch_announcement(ctc_signup)
+    if ctc_signup.email_address.present?
+      CtcSignupMailer.launch_announcement(email_address: ctc_signup.email_address, name: ctc_signup.name).deliver_now
```

Comment:
the choice in the CTC beta email to use deliver_now over deliver_later is something I regret. The text message call queues a job, so using deliver_later here to queue a job may also make sense.